### DISCUSSION TITLE
what's with the low detection threshold?

### DISCUSSION THREAD

**Discussion by dedepene** [blakeblackshear/frigate#3693] [General]
Hello!

I'm struggling to understand why detection seems to be triggered at such low threshold (62% in the screen with the bounding box), while i have "detect" configured above 80 in the frigate config on a home assistant addon. Ideas please? 

![fr2](https://user-images.githubusercontent.com/6883390/185881111-712d8d86-f108-4b01-a7c6-aad9227b0d5f.png)
![fr1](https://user-images.githubusercontent.com/6883390/185881117-de6c4479-a4e4-4895-b845-32d4c6dcc9d5.png)

**Comment by NickM-27** [comment]
You can see in the events page the high score was 82. 

See https://docs.frigate.video/guides/false_positives for how scores are calculated. 

Once an object is a true positive it will remain one. In this case spider webs seem to really mess with the google models for whatever reason.
